classes/autoptimizeConfig.php

```diff
@@ -8,10 +8,22 @@ class autoptimizeConfig {
 	//Singleton: private construct
 	private function __construct() {
 		if( is_admin() ) {
+			
+			// Load custom admin styles
+			add_action( 'admin_enqueue_scripts', array( $this, 'load_custom_wp_admin_style' ) );
```

Comment:
can't we do this in autoptimizeToolbar.php? that way we avoid styles being loaded if not needed?